assisted-events-scrape/dictpath/dictpath.py

```diff
@@ -0,0 +1,58 @@
+from typing import List
```

Comment:
It's very similar to https://pypi.org/project/path-dict/ no?
If so, I prefer using that external project (or anything alike)

Reply:
There is https://github.com/dpath-maintainers/dpath-python
Just reading a bit through it, it seems to support lists and filtering/deletion

Reply:
I think this one would do, thanks for finding it!! I'll update this PR

Reply:
Yeah looks great
Thanks!